### PR TITLE
Add work for first challenge and styled components

### PR DESCRIPTION
**Pull request by Ferleomos**
This is the work for the first React challenge.

package.json not committed, add styled-components package when testing it.

### CODE REVIEW

File: src/components/App/App.component.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { useLayoutEffect } from 'react';
+//import React, { useLayoutEffect } from 'react';
```
Comment (ragrum) [inline src/components/App/App.component.jsx]: Do not keep commented out code, that's what your Git history is for.

File: src/components/App/App.component.jsx [hunk 2]
```diff
@@ -7,31 +8,33 @@ import LoginPage from '../../pages/Login';
...
+// import Fortune from '../Fortune';
 import Layout from '../Layout';
-import { random } from '../../utils/fns';
+//import { random } from '../../utils/fns';
+import HeaderCotnainer from '../Header'
 
 function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
+  // useLayoutEffect(() => {
```
Comment (ragrum) [inline src/components/App/App.component.jsx]: Ditto.

File: src/components/Container/CardContainer.component.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    margin: 0px auto;
+    flex: 1 1 0%;
+    display: flex;
+    flex-flow: row wrap;
+    border: none;
+    justify-content: center;
+`;
+
+function VideoContainer(){
+    // console.log(mockYoutubeData);
```
Comment (ragrum) [inline src/components/Container/CardContainer.component.jsx]: Remove commented out code.

File: src/components/Header/MenuButton.component.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    @media(max-width: 640px) {
+        margin: 0px 0px 0px 10px;
+    }
+`;
+
+function MenuIcon(){
+    return(
+        <Button>
+            <Icon viewBox="0 0 24 24">
+                <path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"></path>
```
Comment (ragrum) [inline src/components/Header/MenuButton.component.jsx:28]: You could use [SVGr](https://react-svgr.com/) to convert SVGs to React components. No change is required as this as this is a preference.

**Review by ragrum — CHANGES_REQUESTED** [review]
Remove all commented out code, please. Ping me on Slack when done. 🙂

File: src/components/App/App.component.jsx (hunk 1 above)
Comment (Ferleomos) [reply src/components/App/App.component.jsx]: Done.

File: src/components/App/App.component.jsx (hunk 2 above)
Comment (Ferleomos) [reply src/components/App/App.component.jsx]: Done

File: src/components/Container/CardContainer.component.jsx (hunk 3 above)
Comment (Ferleomos) [reply src/components/Container/CardContainer.component.jsx]: Done. thanks.

File: src/components/Header/MenuButton.component.jsx (hunk 4 above)
Comment (Ferleomos) [reply src/components/Header/MenuButton.component.jsx:28]: I'll take a look and implement it in future changes. :D

File: src/components/App/App.component.jsx [hunk 5]
```diff
@@ -7,31 +7,13 @@ import LoginPage from '../../pages/Login';
 import NotFound from '../../pages/NotFound';
 import SecretPage from '../../pages/Secret';
 import Private from '../Private';
-import Fortune from '../Fortune';
 import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import HeaderCotnainer from '../Header'
```
Comment (ragrum) [inline src/components/App/App.component.jsx]: Typo in `HeaderCotnainer`

**Review by ragrum — APPROVED** [review]
Just found a typo, you can fix it and merge. Nice work! 👍

File: src/components/App/App.component.jsx (hunk 5 above)
Comment (Ferleomos) [reply src/components/App/App.component.jsx]: fixed and merged.
thanks

**Pull request merged**